sdk/core/abort-controller/README.md

```diff
@@ -1,61 +1,81 @@
-# Azure Core Aborter library for JS
+# Azure Abort Controller library for JS
 
-`Aborter` is used by the Azure SDK for JavaScript to abort pending work based on timeouts or other signals. `Aborter` is compatible with the `AbortSignal` used by the `fetch` API built into modern browsers.
+The `@azure/abort-controller` package provides `AbortController` and `AbortSignal` classes that are compatible
+with the `AbortController` built into modern browsers and the `AbortSignal` used by `fetch`.
+These classes are provided for use with APIs in the Azure SDK for JavaScript that accept an `AbortSignalLike`
+parameter to cancel an operation.
 
 ## Getting started
 
 ### Installation
- - Installing this library
+
+- Installing this library
+
 ```
 npm install @azure/abort-controller
 ```
 
 ## Key Concepts
-This library contains the `Aborter` class which is used by the Azure SDK for JavaScript to abort pending
-work based on timeouts or other signales.
-`Aborter` is compatible with the `AbortSignal` used by the `fetch` API built into modern browsers.
+
+Use the `AbortController` to create an `AbortSignal` which can then be passed to Azure SDK operations to cancel
+pending work. The `AbortSignal` can be accessed via the `signal` property on an instantiated `AbortController`.
+An `AbortSignal` can also be returned directly from a static method, e.g. `AbortController.timeout(100)`.
+that is cancelled after 100 milliseconds.
+
+Calling `abort()` on the instantiated `AbortController` updates the associated `AbortSignal`.
+Any subsequent calls to `abort()` on the same controller will have no effect.
+
+The `AbortSignal.none` static property returns an `AbortSignal` that can not be aborted.
+Provde this to an Azure SDK method that requires an `abortSignal` if you don't wish to add custom abort logic.
```

Comment:
We have methods that have the abortSignal to be mandatory??

Reply:
If we update the storage APIs to use `abortSignal`, then yes.
https://github.com/Azure/azure-sdk-for-js/blob/d6201b5e7b0ecda13d44fe97c59b370311ce16a1/sdk/storage/storage-blob/src/highlevel.node.ts#L43

Most (all?) of these APIs require the 1st parameter to be an aborter. I'd like if this wasn't the case though.

Reply:
Turns out Track 2 Storage dont need the abortSignal to be mandatory, so we dont need this anymore? Confirm with @bterlson that there are no other use cases for `AbortSignal.none`

Reply:
The primary use case for AbortSignal.none is when writing your own code that takes an AbortSignal and you need a default value. Consider something like:

```
async function uploadFile(abortSignal = AbortSignal.none) {
  ...
}
```